mico-core/src/main/java/io/github/ust/mico/core/REST/ServiceController.java

```diff
@@ -264,9 +267,14 @@
         log.debug("Start importing MicoService from URL '{}'", url);
         RestTemplateBuilder restTemplate = new RestTemplateBuilder();
         GitHubCrawler crawler = new GitHubCrawler(restTemplate);
-        MicoService newService = crawler.crawlGitHubRepoLatestRelease(url);
-
-        return createService(newService);
+        try {
+            MicoService newService = crawler.crawlGitHubRepoLatestRelease(url);
+            return createService(newService);
+        } catch (IOException e) {
+            e.printStackTrace();
+            log.error("Getting exception '{}'", e.getMessage());
+            throw new ResponseStatusException(HttpStatus.INTERNAL_SERVER_ERROR, e.getMessage(), e);
```

Comment:
Delivering the whole exception to the API might not be an good idea, since a simple message should be enough.